apm-agent-plugins/apm-servlet-plugin/src/test/java/co/elastic/apm/agent/servlet/ServletInstrumentationTest.java

```diff
@@ -126,6 +145,13 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
         }
     }
 
+    public static class EnsureInitServlet extends HttpServlet {
+        @Override
+        protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException {
+            init(getServletConfig());
```

Comment:
Why not override the `init` method?

Reply:
What would be the benefit of that?

Reply:
The fact that it is how it is going to be invoked in reality, by the servlet container, so probably a better way to test this.. 
However, it means that `ServletInstrumentationTest#initInstrumentation` will need to use the same tracer object across tests, so only if this is not a problem.

Reply:
done